lib/msal-react/docs/migration-guide.md

```diff
@@ -0,0 +1,341 @@
+# Migration Guide from msal v1 to @azure/msal-react and @azure/msal-browser
+
+***
+
+1. [Updating your app registration](#updating-your-app-registration)
+1. [Installation](#installation)
+1. [Upgrading from react-aad-msal](#upgrading-from-react-aad-msal)
+    1. [Breaking Changes](#breaking-changes)
+    1. [Initialization](#initialization)
+    1. [Protecting your components](#protecting-your-components)
+    1. [Acquiring an access token](#acquiring-an-access-token)
+    1. [Acquiring an id token](#acquiring-an-id-token)
+    1. [Updating redux store integration / reacting to events](#updating-redux-store-integration-/-reacting-to-events)
```

Comment:
Just double checking if this link works. In the rich diff display, it doesn't show up as a link

Reply:
Thanks! Fixed.